### PR TITLE
Typo and formatting fixes

### PR DESCRIPTION
**Pull request by jensmaurer**
Partially addresses #5428

### CODE REVIEW

File: source/utilities.tex [hunk 1]
```diff
@@ -12349,7 +12349,7 @@
 
 \indexlibraryctor{default_searcher}%
 \begin{itemdecl}
-constexpr default_searcher(ForwardIterator pat_first, ForwardIterator pat_last,
+constexpr default_searcher(ForwardIterator1 pat_first, ForwardIterator1 pat_last,
```
Comment (jwakely) [inline source/utilities.tex:12352]: Should we change the synopsis to match?

Comment (jensmaurer) [reply source/utilities.tex:12352]: It's correct in the synopsis. (ForwardIterator1 is a template parameter of the enclosing class.)

Comment (jwakely) [reply source/utilities.tex:12352]: Right, but in the synopsis it's just called `ForwardIterator`. It's not incorrect, just inconsistent.

Comment (jensmaurer) [reply source/utilities.tex:12352]: You were talking about the header synopsis; I was looking at the class synopsis (and found not issue).

The header synopsis is now fixed.

Comment (jwakely) [reply source/utilities.tex:12352]: Yes, sorry for being unclear! I should have said header.

**Review by jwakely — APPROVED** [review]

**Comment by jensmaurer** [comment]
@jwakely , I've pushed more fixes. Please approve again. I'll then merge and continue in a fresh pull request.

**Review by jwakely — APPROVED** [review]

**Comment by jensmaurer** [comment]
@tkoeppe, I have no idea. My guess is that it happens only if we have a trailing underscore, so it's likely related to the underscore package.

**Pull request merged**